airflow/sentry.py

```diff
@@ -81,13 +81,17 @@ def __init__(self):
             sentry_celery = CeleryIntegration()
             integrations.append(sentry_celery)
 
-        dsn = conf.get("sentry", "sentry_dsn")
+        dsn = None
+        sentry_config_opts = conf.getsection("sentry") or {}
```

Comment:
Can you add this option to reference documentation?
airflow/config_templates/config.yml
https://airflow.readthedocs.io/en/latest/configurations-ref.html

Reply:
This section has already existed in docs. I update section description in airflow/config_templates/config.yml with info of supporting additional configs from sentry docs.